src/commands/context/tasks.ts

```diff
@@ -0,0 +1,104 @@
+ * limitations under the License.
+ *
+ */
+import {
+  Task, Templates
+} from '../../core/index.js'
+import * as flags from '../flags.js'
+import type { ListrTaskWrapper } from 'listr2'
+import { type BaseCommand } from '../base.js'
+
+export class ContextCommandTasks {
+
+  private readonly parent: BaseCommand
+  private readonly promptMap: Map<string, Function>
+
+  constructor (parent, promptMap) {
+    this.parent = parent
+    this.promptMap = promptMap
+  }
+
+  updateLocalConfig (argv) {
+    return new Task('Update local configuration', async (ctx: any, task: ListrTaskWrapper<any, any, any>) => {
+      this.parent.logger.info('Updating local configuration...')
+
+      const isQuiet = !!argv[flags.quiet.name]
+
+      let currentDeploymentName = argv[flags.namespace.name]
+      let clusterAliases = Templates.parseClusterAliases(argv[flags.clusterName.name])
+      let contextName = argv[flags.context.name]
+
+      const kubeContexts = await this.parent.getK8().getKubeConfig().getContexts()
+
+      if (isQuiet) {
+        const currentCluster = (await this.parent.getK8().getKubeConfig().getCurrentCluster())
+        if (!clusterAliases.length) clusterAliases = [currentCluster.name]
+        if (!contextName) contextName = await this.parent.getK8().getKubeConfig().getCurrentContext()
+
+        if (!currentDeploymentName) {
+          const selectedContext = kubeContexts.find(e => e.name === contextName)
+          currentDeploymentName = selectedContext && selectedContext.namespace ? selectedContext.namespace : 'default'
+        }
+      }
+      else {
+        if (!clusterAliases.length) {
+          const prompt = this.promptMap.get(flags.clusterName.name)
+          const unparsedClusterAliases = await prompt(task, clusterAliases)
+          clusterAliases = Templates.parseClusterAliases(unparsedClusterAliases)
+        }
+        if (!contextName) {
+          const prompt = this.promptMap.get(flags.context.name)
+          contextName = await prompt(task, kubeContexts.map(c => c.name),  contextName)
+        }
+        if (!currentDeploymentName) {
+          const prompt = this.promptMap.get(flags.namespace.name)
+          currentDeploymentName = await prompt(task, currentDeploymentName)
+        }
+      }
+
+      // Select current deployment
+      this.parent.getLocalConfig().setCurrentDeployment(currentDeploymentName)
+
+      // Set clusters for active deployment
+      const deployments = this.parent.getLocalConfig().deployments
+      deployments[currentDeploymentName].clusterAliases = clusterAliases
+      this.parent.getLocalConfig().setDeployments(deployments)
+
+      this.parent.getK8().getKubeConfig().setCurrentContext(contextName)
+
+      this.parent.logger.info(`currentDeploymentName: ${currentDeploymentName}`)
+      this.parent.logger.info(`contextName: ${contextName}`)
+      this.parent.logger.info(`clusterAliases: ${clusterAliases.join(' ')}`)
+      this.parent.logger.info('Save LocalConfig file')
```

Comment:
```suggestion
      this.parent.logger.info('Save LocalConfig file: [currentDeploymentName: ${currentDeploymentName}, contextName: ${contextName}, clusterAliases: ${clusterAliases.join(' ')}]')
```
less verbose in the log file if it is on one line